packages/contracts/evm-contracts/contracts/token/IERC4906Agnostic.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+// Forked from OpenZeppelin Contracts (last updated v5.0.0) (interfaces/IERC4906.sol)
+
+pragma solidity ^0.8.13;
+
+/// @title Agnostic Metadata Update Extension
+interface IERC4906Agnostic {
```

Comment:
We didn't use this for the inverse projected ERC721. Maybe we should switch to using it there as well?

Reply:
The reason we use this here is that the OpenZeppelin's implementation of the `IERC4906` interface extends the `IERC721` interface, so we cannot use it but have to have this custom one.